tests/pure_cpp/smart_holder_poc_test.cpp

```diff
@@ -37,6 +37,16 @@ struct indestructible_int {
     ~indestructible_int() = default;
 };
 
+struct base {
+    virtual int get() { return 10; }
+    virtual ~base() {}
+};
+
+struct derived : public base {
+    int get() override { return 100; }
+    ~derived() override {}
```

Comment:
Could we delete this override?
(It seems unusual to override the virtual dtor in this situation.)